brain/python/CMakeLists.txt

```diff
@@ -12,17 +13,32 @@ if( NOT Boost_PYTHON${USE_BOOST_PYTHON_VERSION}_LIBRARY OR
 endif()
 
 set(BRAIN_PYTHON_SOURCE_FILES
+  arrayHelpers.cpp
   brain.cpp
+  circuit.cpp
   spikeReportWriter.cpp
   spikeReportReader.cpp
-  spikes.cpp)
+  spikes.cpp
+  synapses.cpp
+  submodules.cpp
+  neuron/module.cpp
+  neuron/morphology.cpp
+)
+if(BBPTESTDATA_FOUND)
```

Comment:
We have plenty of places where FOO_FOUND is used, why is your suggestion better.